km_api/know_me/serializers/media_resource_serializers.py

```diff
@@ -26,3 +26,22 @@ class Meta:
         }
         fields = ('id', 'url', 'name', 'file', 'permissions')
         model = models.MediaResource
+
+
+class MediaResourceCategorySerializer(
+        serializers.HyperlinkedModelSerializer):
+    """
+    Serializer for ``MediaResourceCategory`` instances.
+    """
+    permissions = DRYPermissionsField()
+    url = serializers.HyperlinkedIdentityField(
+        view_name='know-me:media-resource-category-list')
```

Comment:
The URL is traditionally used for the item's detail view. Is there a reason for using the list view instead?